index.js

```diff
@@ -127,12 +137,19 @@ function wkhtmltopdf(input, options, callback) {
   var stderrMessages = [];
   function handleError(err) {
     var errObj = null;
+    var code;
+    var parallelError;
     if (Array.isArray(err)) {
+      code = err.code;
+      parallelError = err.parallelError;
+      // fix cutted lines in Windows
+      err = Buffer.concat(err).toString();
```

Comment:
What does this fix?

Reply:
comments changed
      // as `err` could contain a small chunks of stdout (and it does sometimes in Windows)
      // we have to concatenate it before using